sds_data_manager/lambda_code/SDSCode/queries.py

```diff
@@ -36,4 +77,14 @@ def lambda_handler(event, context):
     # search the opensearch instance
     search_result = client.search(query, Index(os.environ["OS_INDEX"]))
     logger.info("Query Search Results: " + json.dumps(search_result))
-    return search_result
+
+    # Format the response
+    response = {
```

Comment:
Not saying this should happen in this PR, but would it make sense to create a Response class with different formatted responses that we could instantiate with body messages, so we can reuse these instead creating these dictionaries in the lambdas?